front/app/components/Sharing/SharingDropdownButton/SharingDropdownContent.tsx

```diff
@@ -173,15 +190,34 @@ const SharingDropdownContent = ({
     </Email>
   );
 
+  const copylink = (
+    <CopyLink
+      setLinkCopied={setLinkCopied}
+      copyLink={url}
+      className={`sharingButton copylink`}
+    >
+      <>
```

Comment:
```
<CopyLinkIcon ariaHidden name="link" />
        {!linkIsCopied && (
          <span aria-hidden>{formatMessage(messages.shareByLink)}</span>
        )}
        {linkIsCopied && (
          <span aria-hidden>{formatMessage(messages.linkCopied)}</span>
        )}
```
This should all be part of the CopyLink component, if this is the way we want to structure it..